### PR TITLE
Bugs/components/pages/parent faq

### PR DESCRIPTION
**Pull request by just-kato**
## Description

Add a summary that address the following:

-With this ticket I updated the styling on the Parent FAQ page. All of the contents regarding the the question and answer cards were shifted to the left. I created .ant-card.card and removed the .card because the ,card was not selecting all of the cards contains. Once I did this I was able to adjust the border-radius to match the faq container above. I also created .site-card-border-less-wrapper in order to center the cards to the middle of the screen.

## Commits Checklist

- [x] added .site-card-border-less-wrapper to move center content.
- [x] removed extra spacing
- [x] changed the h4 tag to h3 so the font would be bigger

## Code Checklist

[x] Adjust padding for first question of each subsection

[x] Center the Game Play + Content container

https://trello.com/c/Dpgxdm3d/647-import-parentfaq-from-components-pages-parentfaq

### CODE REVIEW

File: src/styles/less/ParentFaq.less [hunk 1]
```diff
@@ -9,18 +9,31 @@
 .parent-back-button {
   .parent-back-button();
 }
+.site-card-border-less-wrapper{
+  display: flex;
+  justify-content: center;
+}
 
-.card {
+.ant-card.card {
```
Comment (Julianisrel) [inline src/styles/less/ParentFaq.less:17]: Good job, correcting the styles with ant-design.

File: src/styles/less/ParentFaq.less [hunk 2]
```diff
@@ -9,18 +9,31 @@
...
   width: 66%;
   background-color: #fbfbfb;
   position: relative;
   margin: 25px auto;
   padding: 20px;
-  border-radius: 16px;
+  border-radius: 1.5%;
```
Comment (OniWarrior) [inline src/styles/less/ParentFaq.less:23]: Good job on the radius. Switching to percentage versus hardcoded pixels will make this more responsive.

**Review by OniWarrior — APPROVED** [review]
Good work on this ticket. The changes to ant design will keep all of our components and stylings consistent to what Jake wants to see in the codebase.

**Review by Veilios — APPROVED** [review]
Everything looks clean and concise! Awesome work!

**Pull request merged**